conozco.py

```diff
@@ -270,7 +270,7 @@ def loadInfo(self):
                 lugares = lugares + f.HILLS
             self.listaLugares = list()
             for c in lugares:
-                nombreLugar = unicode(c[0], 'UTF-8')
+                nombreLugar = str(c[0], 'UTF-8')
```

Comment:
Is this conversion necessary?  Also the others like it.  They seem without purpose.